tests/AutoReview/TravisTest.php

```diff
@@ -113,11 +113,9 @@ private function getAllPhpVersionsUsedByCiForDeployments()
             return 'Deployment' === $job['stage'];
         });
 
-        $versions = array_map(function ($job) {
+        return array_map(function ($job) {
```

Comment:
Shouldn't this be handled by `return_assignment`?

Reply:
could be, maybe best to open a new ticket on it though